_posts/2019-08-28-mentor-spotlight-katharine-hyatt.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Mentor Spotlight: Katharine Hyatt"
+date: 2019-08-31 17:00:00
+categories: spotlights
+tags: HackIllinois2019, mentors, featured
```

Comment:
Learned this the hard way: `featured` should be the first tag out of these. Also, don't forget to remove `featured` as a tag from the previous blog post.